10x_Experimental_Scenarios.md

```diff
@@ -7,87 +7,124 @@ The relationship between **biological samples** (e.g. a cell suspension extracte
-
-### Sample-Level Spreadsheet
-| Sample name  | Loading Sample |  Sample-Meta-Data ... | HTO | Expected Cell Number | Library Features | 
-|---|---|---|---|---|--|
-| S1_GEX  | H1_GEX | ... | HTO-1  | 5000 | LF-1 |
-
-A CITE-seq run sequencing-library will, in general, contain ADTs and HTOs. The list of features assocaited with this sequencing library are stored in a separate `Library Features Table` (see below and foreign keys under the `Library Features` column). For now, we are omitting the `Sample-Meta-Data` which will include Cell Ranger necessary information like organism, reference transcriptome etc. 
-
-### Library Features Table
-| Library Features| Index |
-|---|---|
-| HTO-1  | HTO-Index-1 |
-| CD8_HIMC-1_Lot-1  | ADT-Index-1 |  
+- `FASTQs`: name of the "set of FASTQs" that will be fed to a single `cellranger count` run.
+  - Composed of the `Loading Sample` name appended with the `BCL Run ID`.
+  - Allows us to handle the common scenario where the same sequencing library (e.g. tube of liquid that can be aliquoted from) is sequenced more than once.
+- `Loading Sample`: name of the sample that is loaded into a 10x chip lane (can consist of several samples via hashing).
+- `Hashed Sample`: True/False, indicates whether hashing has been done (will be redundant with the `Loading Sample` naming convention).
+- `10x Lane ID`: lane number a sample is loaded into
+  - necessary for situations where the same sample is loaded into several lanes
+  - (becomes part of the FASTQs' names in that case)
+- `Library Type`: type of library being prepared (e.g. GEX, ADT)
+  - Chemistry and version may or may not be included, e.g. GEX_5-prime.
+- `BCL Run ID`: name of the BCL file the FASTQs will be put into.
+- `Processing Run`: name of the ["processing run"][`Processing-Run`] that the data is being organized under (a processing run consists of the jobs necessary to convert BCL inputs into FBM and TCR/VDJ outputs).
+
+## Sample-Level Spreadsheet
+| Sample Name  | Loading Sample |  Sample Metadata ... | HTO | Expected Cell Number | Library |
+|---|---|---|---|---|---|
+| S1_GEX  | H1_GEX | ... | HTO-1  | 5000 | L-1 |
+
+<!-- Difference b/t Sample name and Loading sample? -->
+A CITE-seq sequencing-library will, in general, contain ADTs and HTOs. <!-- Should there be an ADTs column? -->
+- `Sample Metadata`: will include necessary Cell Ranger information like organism, reference transcriptome, etc.
+- `Library`: link to the [Library Features Table] below.
+<!-- other column descriptions? -->
+
+## Library Features Table
+| Library | Feature | Index |
+|---|---|---|
+| L-1 | HTO-1 <!-- is this supposed to match the HTO column above? -->  | HTO-Index-1 |
```

Comment:
Slightly confusing example value `HTO-1` matches a column above that it doesn't appear to be a foreign key of.

Reply:
I discussed that here (https://github.com/ismms-himc/cellranger-aws-pipeline-rebuild/pull/2#discussion_r322779085) we're trying to link to the entire list of features. Each experiment will have a different set of ADTs (e.g. surface marker proteins that are being measured) and/or HTOs (labels for different samples that are being hashed together).